fashion-ai-repo/Makefile

```
@@ -0,0 +1,189 @@
+	$(PYTHON) -m pytest $(TEST_DIRS) -v
+
+test-cov:
+	@echo "$(BLUE)Running tests with coverage...$(RESET)"
+	$(PYTHON) -m pytest $(TEST_DIRS) --cov=$(SRC_DIRS) --cov-report=html --cov-report=term-missing
+	@echo "$(GREEN)Coverage report generated in htmlcov/$(RESET)"
+
+# ============================================================================
+# SECURITY
+# ============================================================================
+
+security:
```

checkmake (0.2.2)

[warning] Target body for "security" exceeds allowed length of 5 (10).

(maxbodylength)